src/components/organisms/UiList/UiList.vue

```diff
@@ -63,13 +99,41 @@ export interface ListProps {
    * Use this props to pass list items.
    */
   items?: ListItem[];
+  /**
+   * Use this props to show button.
+   */
+  showButton?: boolean;
+  /**
+   * Use this props to set button into loading state.
+   */
+  isLoading?: boolean;
+  /**
+   * Use this props to pass attrs for UiLoader.
+   */
+  loaderAttrs?: LoaderAttrsProps;
```

Comment:
For me features in the UiList component are redundant. Maybe wee need a new component with this functionality but UiList must be a very simple component.